src/main/java/mcp/mobius/waila/util/ModInfo.java

```diff
@@ -27,7 +28,11 @@ public static void register(ModInfo info) {
     }
 
     public static ModInfo get(String namespace) {
-        return CONTAINER_CACHE.computeIfAbsent(namespace, s -> ICommonService.INSTANCE.createModInfo(namespace).orElse(new ModInfo(false, s, s, "unknown")));
+        return CONTAINER_CACHE.computeIfAbsent(namespace, s ->
+            ICommonService.INSTANCE.createModInfo(namespace)
```

Comment:
Move this line to the right beside the lambda arrow, also change the `namespace` inside the lambda to use `s` instead.

Reply:
stabbed at it on a phone, let me know if I got it

Reply:
Looks good.